{"name": "Проверка, что система возвращает ошибку при неверном пароле. POST /api/v1/courier/login", "status": "passed", "start": 1759660375827, "stop": 1759660376280, "uuid": "4f3409c8-7001-4318-9667-dcb2b6cc94fc", "historyId": "04f7fe25f3a31f453616e04c20bbf4c9", "testCaseId": "04f7fe25f3a31f453616e04c20bbf4c9", "fullName": "tests.test_login_courier.TestsCourierLogin#test_login_wrong_password", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestsCourierLogin"}, {"name": "host", "value": "Natalias-MacBook-Air.local"}, {"name": "thread", "value": "35525-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}], "titlePath": ["tests", "test_login_courier.py", "TestsCourierLogin"]}